docs/test/live-unit-testing-faq.md

```diff
@@ -51,6 +51,16 @@ Live Unit testing may not work for one of the following reasons:
     </RunSettings> 
    ``` 
 
+## Why does Live Unit Testing show incorrect coverage after upgrading referenced Visual Studio test adapter in projects to supported version?
+
+- If there is multiple projects in the solution reference test adapter NuGet package, all of them need to be upgraded to the supported version.
+
```

Comment:
This FAQ is currently not up-to-date with what's in the blob. @rpetrusha is going to bring it to par with FAQ and then you can update this PR. I think the blog post mentions the first part of this FAQ.